java-grupa8/homework-2/src/sort.java

```diff
@@ -0,0 +1,35 @@
+/**
+ Implements a bubble sort algorithm to sort an array of SalesRepresentative objects in descending order.
+ The sorting should be done based on the amount of revenue generated via sales.
+ Each rep has a number of sales, and a quota / sale.
+ */
+public class sort {
+    public static void main(String[] args) {
+            String[] nameRepr = {"Ion Dumbraveanu", "Alina Iriminescu", "Ovidiu Zaharescu", "Nicoleae Popovici", "Andeea Maxim"};
+            int[] numberSales = {10, 30, 20, 51, 20};
+            int[] quotaSales = {500, 250, 400, 100, 450};
+            int[] revenue = new int[5];
+            for (int a = 0; a < nameRepr.length; a++) {
+                revenue[a] = numberSales[a] * quotaSales[a];
+            }
+            int tempValue = 0;
```

Comment:
As the requirements state: Main class and algorithm classes separated.
The Bubble sort algorithm should be extracted in a separate method or class.